cmd/server/assets/login/registerphone.html

```diff
@@ -56,10 +69,19 @@ <h1>SMS auth registration</h1>
       });
 
     $(function () {
+      let $displayName = $('#display')
```

Comment:
semicolons to pray thee the javascript lords

Reply:
Javascript feels really willy-nilly about stuff. I'm probably spoiled.